package.json

```diff
@@ -36,7 +36,7 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "bleno": "~0.1.13",
+    "bleno": "git+https://github.com/jacobrosenthal/bleno#platform-on-state-change",
```

Comment:
This needs to get fixed, I need a to get platform check seperated from websocket stuff, unless we can just get that all merged